app/views/shared/forms/_savings_and_investments_form.html.erb

```diff
@@ -19,8 +19,8 @@
 
       <div class="govuk-checkboxes">
         <div class="govuk-checkboxes__item">
-          <%= form.check_box('check_box_none_selected', { class: 'govuk-checkboxes__input' }, 'true', false) %>
-          <%= form.label('check_box_none_selected', none_selected, class: 'govuk-label govuk-checkboxes__label') %>
+          <%= form.check_box :none_selected, { class: 'govuk-checkboxes__input' }, 'true', '' %>
```

Comment:
The final parameter being passed in is `''`. Should that be dependent on the current status of `none_selected`? That is, should it be `@none_selected` or `form.none_selected`

Reply:
Using `''` makes it work well but it does seem weird. Using `@none_selected` didn't work completely well with the select/unselect Javascript bit because of the hidden input added by rails.
I modified this code to use `@form.none_selected` and also modified the Javascript to also affect the hidden input.